src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/HttpProxyCacheableRequestGroup.java

```diff
@@ -89,32 +89,33 @@ void onNonCacheableResponse(
 
     void onCacheableResponseUpdated()
     {
-        this.sendSignalToQueuedInitialRequestSubscribers(CACHE_ENTRY_UPDATED_SIGNAL);
+        routeIdsByReplyId.forEach(this::doSignalQueuedInitialRequestSubscribers);
     }
 
     void onCacheableResponseAborted()
     {
         this.routeIdsByReplyId.forEach(this::doSignalCacheEntryAborted);
     }
 
-    private void doSignalCacheEntryAborted(long acceptReplyId, long acceptRouteId)
+    private void doSignalCacheEntryAborted(
+        long acceptReplyId,
+        long acceptRouteId)
     {
         writer.doSignal(acceptRouteId,
                         acceptReplyId,
                         factory.supplyTrace.getAsLong(),
                         CACHE_ENTRY_ABORTED_SIGNAL);
     }
 
-    private void sendSignalToQueuedInitialRequestSubscribers(
-        long signal)
+    private void doSignalQueuedInitialRequestSubscribers(
```

Comment:
Rename to `doSignalCacheEntryUpdated` to match signal name.

Reply:
Done.